proto/eth/service/node_service.proto

```diff
@@ -13,54 +13,54 @@
 // limitations under the License.
 syntax = "proto3";
 
-package ethereum.eth.v1;
+package ethereum.eth.service;
 
 import "google/api/annotations.proto";
 import "google/protobuf/descriptor.proto";
 import "google/protobuf/empty.proto";
 
 import "proto/eth/v1/node.proto";
 
-option csharp_namespace = "Ethereum.Eth.V1";
-option go_package = "github.com/prysmaticlabs/prysm/proto/eth/v1";
+option csharp_namespace = "Ethereum.Eth.Service";
+option go_package = "github.com/prysmaticlabs/prysm/proto/eth/service";
 option java_multiple_files = true;
-option java_outer_classname = "NodeServiceProto";
```

Comment:
again don't think the naming here is correct

Reply:
again thanks for being observant